gems/sorbet/bin/srb-rbi

```diff
@@ -73,7 +102,7 @@ actions:
       STDOUT.write(emojify("❔", "Would you like to continue? [Y/n] "))
       if STDIN.isatty && STDOUT.isatty
         begin
-          input = Kernel.gets&.strip
+          input = STDIN.gets&.strip
```

Comment:
Had to change because of [this](https://stackoverflow.com/a/10523615)

Reply:
Just ran into this issue myself running `srb rbi update`. If this PR gets delayed I'd love to see this change as a separate PR.

Reply:
I now see there's another PR addressing this as well. https://github.com/sorbet/sorbet/pull/1066